packages/web-styles/src/components/card.stories.js

```diff
@@ -0,0 +1,116 @@
+import './card.scss';
+
+export default {
+    title: 'Components/Card',
```

Comment:
This file uses 4 spaces for indentation, but only 2 should be used. Press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> -> Format document in Visual Studio Code to format it correctly.